### PR TITLE
Translated index file

### PR DESCRIPTION
**Pull request by ignotus666**

### CODE REVIEW

File: es-index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+--- title: "Jamulus ‒ Toca música online. Con amig@s. Gratis." lang: "es" permalink: / layout: mainhomepage ldjAppCategory: "Music" ldjSlogan: "Toca música online. Con amig@s. Gratis." ldjCreator: "Volker Fischer y contribuidores" metadescription: "Jamulus es software de código abierto que permite a músicos actuar, ensayar o simplemente hacer jam sessions, en tiempo real a través de Internet." mAltProgIcon: "Jamulus icon" mTSlogan: "Toca música online. Con amig@s. Gratis." mTGetStartedNow: "¡Empieza ya!" mHTMLDownloadNow: 'Avanzado: <a href="https://sourceforge.net/projects/llcon/files/latest/download" target="_blank" id="dld_frm_sf" rel="noreferrer">descarga ahora</a>' ---
```
Comment (ann0see) [inline es-index.html:1]: I'm not sure if you can do it all in one line?

**Review by ann0see — COMMENTED** [review]
I think there might be a problem with this pr since in index.html you didn't start a new line in the front matter

File: es-index.html (hunk 1 above)
Comment (ignotus666) [reply es-index.html:1]: Yep, my translation tool messed up the formatting - should be corrected now.

Comment (ann0see) [reply es-index.html:1]: Ok. The last `---` should probably also be in a new line.

Comment (ignotus666) [reply es-index.html:1]: Yes, you're right... fixed!

**Comment by ann0see** [comment]
Ok. Seems to be allright now.

**Pull request merged**